docs/themes/smithy/static/default.css_t

```diff
@@ -538,7 +538,7 @@ table.hlist {
 
 code {
   font-size: 90%;
-  background-color: #f2f2f2;
+  background-color: #e9e9e9;
```

Comment:
Slightly darker to be visible in dark background table rows without being obnoxious in light backgrounds.